portfolio/src/main/webapp/script.js

```diff
@@ -109,20 +109,23 @@ $(document).ready(() => {
  * Adds comments to the page.
  */
 function addComments() {
-  fetch('/data').then(response => response.json()).then((messages) => {
+  console.log("Hello!");
```

Comment:
looks like you've got two leftover console logs here

Reply:
Thanks for the catch, removed them.